cl/recap/mergers.py

```diff
@@ -1670,6 +1668,7 @@ def save_iquery_to_docket(
     d: Docket,
     tag_names: Optional[List[str]],
     add_to_solr: bool = False,
+    avoid_trigger_signal=False,
```

Comment:
We're missing the type hint for this parameter.

Reply:
Sure, added it.